docs/products/opensearch/dashboards/howto/access-from-chrome.rst

```diff
@@ -0,0 +1,8 @@
+Accessing Aiven for OpenSearch Dashboards from Chrome
+=====================================================
+
+Access to Aiven for OpenSearch Dashboards requires a valid username and password (HTTP Basic Authentication.)  The service overview page in the `Aiven console <https://console.aiven.io>`_ includes a link directly to OpenSearch Dashboards which embeds the username and password in the URL, allowing one-click access to Kibana from the Aiven console.
```

Comment:
Maybe we need to add a link to this article from the Note in the homepage?

Reply:
I've removed it altogether, seems this was changed. FF and Chrome behave the same and they automatically ask for name and password. I think this article doesn't add any value.